bundles/org.openhab.binding.unifiprotect/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,731 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="unifiprotect"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Bridge: NVR -->
+	<bridge-type id="nvr">
```

Comment:
Please add the semantic equipment tags to the things. 
For reference: https://www.openhab.org/docs/developer/bindings/thing-xml.html#thing-semantic-equipment-tags